react-ui/src/pages/contest/useContestSizing.jsx

```diff
@@ -0,0 +1,63 @@
+/**
+ * Voting and winners
```

Comment:
Update comment?

Reply:
Noticed file comments should be updated in several other files as well.

Reply:
Oh sorry, if I'm refactoring I usually copy and paste the original file / similar file to get all the boilerplate in place. Those are now fixed.